manifests/v/voidtools/Everything/Alpha/1.5.0.1344a/voidtools.Everything.Alpha.installer.yaml

```
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.2.3 $debug=QUSU.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: voidtools.Everything.Alpha
+PackageVersion: 1.5.0.1344a
+Platform:
+  - Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://www.voidtools.com/Everything-1.5.0.1344a.x64-Setup.exe
+    InstallerSha256: 33D04E11CEDB0CBFC6752B23DBDCACE58713D0C6CE01FC4866AEFD21FDB3D2BD
+    ProductCode: Everything 1.5a
+    AppsAndFeaturesEntries:
+      - DisplayName: Everything 1.5.0.1344a (x64)
+        DisplayVersion: 1.5.0.1344a

```

```suggestion
        DisplayVersion: 1.5.0.1344
```
Must match Apps&Features / `appwiz.cpl`